components/organisms/UserSettingsPage/user-settings-page.tsx

```diff
@@ -93,14 +113,26 @@ const UserSettingsPage = ({ user }: userSettingsPageProps) => {
     }
   };
 
-  const handleUpdateProfile = async () => {
+  const handleUpdateProfile = async (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
```

Comment:
Instead of using refs for all the fields, why not just get them from the `e.target.form`?